src/setup/cortx_deploy

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+function log_debug {
+    [[ -z $DEBUG || $DEBUG != 1 ]] && return
+
+    echo "log: $*"
+}
+
+function usage {
+    echo "usage: $(basename $0) -n <node id> [-f <solution conf>]"
```

Comment:
Lets change usage as:
`usage: $(basename $0) -n <NODE-ID> [-c <SOLUTION-CONFIG>]`
`where:`
`NODE-ID: Unique Node ID`
`SOLUTION-CONFIG: Solution-Config directory with config.yaml and cluster.yaml (default: /etc/cortx/solution)`